recipes/glib/all/conanfile.py

```diff
@@ -276,10 +285,11 @@ def package_info(self):
             self.package_folder, "bin", "glib-compile-schemas"
         )
 
-        self.cpp_info.components["gresource"].libs = []  # this is actualy an executable
-        self.cpp_info.components["gresource"].requires.append(
-            "libelf::libelf"
-        )  # this is actualy an executable
+        self.cpp_info.components["gresource"].libs = []  # this is actually an executable
+        if self.options.get_safe("with_elf", True):
+            self.cpp_info.components["gresource"].requires.append(
+                "libelf::libelf"
+            )  # this is actualy an executable
```

Comment:
```suggestion
            )  # this is actually an executable
```
let's correct this typo too ;)